root/static/scripts/edit/externalLinks.js

```diff
@@ -118,32 +124,32 @@ export class ExternalLinksEditor extends React.Component<LinksEditorProps, Links
   }
 
   getEditData() {
-    var oldLinks = this.getOldLinksHash();
-    var newLinks = _.keyBy<
+    const oldLinks = this.getOldLinksHash();
+    const newLinks = _.keyBy<
       LinkStateT,
       $ElementType<LinkStateT, 'relationship'>,
     >(this.state.links, 'relationship');
 
     return {
-      oldLinks: oldLinks,
+      allLinks: _.defaults(_.clone(newLinks), oldLinks),
       newLinks: newLinks,
-      allLinks: _.defaults(_.clone(newLinks), oldLinks)
+      oldLinks: oldLinks,
     };
   }
 
   getFormData(startingPrefix: string,
-              startingIndex: number,
-              pushInput: (string, string, string) => void) {
-    var index = 0;
-    var backward = this.props.sourceType > 'url';
-    var {oldLinks, newLinks, allLinks} = this.getEditData();
+    startingIndex: number,
+    pushInput: (string, string, string) => void) {
```

Comment:
The indentation made more sense before, I think? Now only `startingIndex` is on its own line and `startingPrefix` isn't lined up with the ones below. I guess they should all have their own line if eslint wants that.

```JS
  getFormData(
    startingPrefix: string,
    startingIndex: number,
    pushInput: (string, string, string) => void,
  ) {
```